web/modules/custom/dcamp_schedule/templates/schedule.html.twig

```diff
@@ -42,180 +42,197 @@
       </tr>
       <tr>
         <th scope="row">10:00</th>
-        <td class="english featured basic">
-          <a href="/sessions/putting-yourself-out-there-and-avoiding-what-ifs-mikeherchel"
-             target="_blank">Putting Yourself Out There and Avoiding the “What Ifs” <span
-                class="author"> {{ 'by'|t }} Mike Herchel</span></a>
-          <span class="english"></span>
+        <td class="featured basic">
+          <a href="#"
+             target="_blank">{{ 'Session'|t }} <span
+                    class="author"> {{ 'by'|t }} Preston So</span></a>
           <span class="featured"></span>
-          <span class="basic"></span>
         </td>
-        <td class="intermediate"><a href="/sessions/buenas-practicas-front-end-maquetacion-por-componentes"
-                                    target="_blank">Buenas prácticas front-end: Maquetación por componentes <span
-                    class="author"> {{ 'by'|t }} Cristian Aliaga</span></a>
+        <td class="intermediate"><a href="https://2018.drupalcamp.es/sessions/elasticsearch-101-isholgueras"
```

Comment:
Do we need this session with a full URL? This won't work in other environments than production.